weblab/experiments/views.py

```diff
@@ -45,6 +54,30 @@ class ExperimentsView(TemplateView):
     template_name = 'experiments/experiments.html'
 
 
+class ExperimentTasks(LoginRequiredMixin, ListView):
+    """
+    Show running versions of all experiments
+    Delete checked versions
+    """
+    model = RunningExperiment
+    template_name = "experiments/experiment_tasks.html"
+
+    def get_queryset(self):
+        return RunningExperiment.objects.filter(
+            Q(experiment_version__author=self.request.user)
```

Comment:
You don't actually need the `Q()` wrapper for simple filtering cases like this; you can just do `.filter(experiment_version__author=self.request.user)`.

Reply:
good spot - done